docs/developer-guide/architecture.md

```diff
@@ -5,7 +5,7 @@
 At a high level, there are a few key parts to ESLint:
 
 * `bin/eslint.js` - this is the file that actually gets executed with the command line utility. It's a dumb wrapper that does nothing more than bootstrap ESLint, passing the command line arguments to `cli`. This is intentionally small so as not to require heavy testing.
-* `lib/api.js` - this is the entry point of `require("eslint")`. This file exposes an object that contains public classes `Linter`, `CLIEngine`, `RuleTester`, and `SourceCode`.
+* `lib/api.js` - this is the entry point of `require("eslint")`. This file exposes an object that contains public classes `Linter`, `CLIEngine`, `ESLint`, `RuleTester`, and `SourceCode`.
```

Comment:
`CLIEngine` was removed.
```suggestion
* `lib/api.js` - this is the entry point of `require("eslint")`. This file exposes an object that contains public classes `Linter`, `ESLint`, `RuleTester`, and `SourceCode`.
```

Reply:
Are you sure to remove `CLIEngine`? It's not removed until eslint v8 which is still in beta as far as I see.

Reply:
yes. that PR has been merged, looks we forgot to change here. // cc @mdjermanovic

Reply:
I think we can remove `CLIEngine`, we have already started merging v8 pull requests including documentation change. Also,`CLIEngine` has been already removed from [`nodejs-api docs`](https://github.com/eslint/eslint/blob/master/docs/developer-guide/nodejs-api.md)

Reply:
Yes, we can remove `CLIEngine` now. Good catch!